src/Couchbase.Lite.Shared/API/Database/Database.cs

```diff
@@ -1099,27 +1072,19 @@ public void Save(Couchbase.Lite.Linq.IDocumentModel model)
         public ListenerToken AddChangeListener([@CanBeNull] TaskScheduler scheduler,
```

Comment:
Needs `ObsoleteAttribute`

Reply:
? The Obsolete Attribute is right above.

Reply:
Oh the diff left out just a few lines there so I didn't notice it, sorry.